packages/agents/router/src/config.ts

```diff
@@ -227,7 +227,9 @@ export const getEnvConfig = (
         process.env.NXTP_MESSAGE_QUEUE_URI ||
         configJson.messageQueue?.uri ||
         configFile.messageQueue?.uri ||
-        "amqp://guest@guestlocalhost:5672",
+        "amqp://guest@guest:localhost:5672",
+      limit:
+        process.env.NXTP_MESSAGE_QUEUE_LIMIT || configJson.messageQueue?.limit || configFile.messageQueue?.limit || 25,
```

Comment:
Pls remove NXTP_ from the prefix! I made the changes in the other branch for the other vars but let's make sure new stuff does not use NXTP_. It can just be `MESSAGE_QUEUE_LIMIT`.